### PR TITLE
ruby_array_evals

### PR DESCRIPTION
**Pull request by avihil1**

### CODE REVIEW

**Comment by viebel** [comment]
Did u check that objects are well displayed also.

Can u think of other data types to check?

**Comment by avihil1** [comment]
Ruby hash isn't displayed well..

**Comment by viebel** [comment]
This is because in Opal, ruby objects are not javascript objects.

What do you suggest?

File: resources/public/playground-dbg.html [hunk 1]
```diff
@@ -7,16 +7,16 @@
 <link rel="stylesheet" type="text/css" href="css/codemirror.css">
 </head>
 <body>
-<pre><code class="javascript">
-    function foo (x) {
-    return x + 1;
+<pre><code class="ruby">
+    def foo (x) {
```
Comment (iliabylich) [inline resources/public/playground-dbg.html]: It's not a valid ruby, Opal [can't parse it](http://opalrb.org/try/?code:def%20foo(x)%20%7B%0A%20%20x%20%2B%201%0A%7D%0A%20)

Comment (avihil1) [inline resources/public/playground-dbg.html]: :+1:  was changed locally only... thx

**Comment by viebel** [comment]
Merged into master.
Will be deployed soon.
Thanks.

**Comment by viebel** [comment]
deployed!

Ruby arrays are displayed propertly http://blog.klipse.tech/klipse/2016/09/09/klipse-languages.html


image

**Pull request closed without merging**